api/profile/profileRouter.js

```diff
@@ -172,6 +172,17 @@ router.get('/', function (req, res) {
  *      404:
  *        description: 'Profile not found'
  */
+
+router.get('/getServiceProviders', function (req, res) {
+  Profiles.findServiceProviders()
+    .then((serviceProviders) => {
+      res.status(200).json(serviceProviders);
+    })
+    .catch(() => {
+      res.status(404).json({ error: 'No Service Providers Found' });
+    });
+});
+
```

Comment:
Tested this endpoint locally on Nathan's machine. Worked as intended to return First and Last Name of Service Providers.